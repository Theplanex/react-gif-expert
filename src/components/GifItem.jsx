import React from 'react'
import {PropTypes} from 'prop-types'
 
export function GifItem({url,title,id}) {
  return (
    <div className='card'>
        <img src={url} alt={title}  />
        <p>{title}</p>
    </div>
  )
}


GifItem.propTypes = {
  url: PropTypes.string.isRequired,
  title : PropTypes.string.isRequired
}

export default GifItem