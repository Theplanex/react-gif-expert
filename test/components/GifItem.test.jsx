import { render } from "@testing-library/react"
import GifItem from "../../src/components/GifItem"

const url = "url";
const title = "title";
describe('Probando componente GifItem', () => 
{ 
    test('Prueba de snapshot', () => { 


         const {container} = render(<GifItem title={title} url={url} />)
         expect(container).toMatchSnapshot();
     }) 


})