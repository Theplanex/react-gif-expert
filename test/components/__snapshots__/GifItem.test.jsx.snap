// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Probando componente GifItem Prueba de snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="title"
      src="url"
    />
    <p>
      title
    </p>
  </div>
</div>
`;
